keyboards/rad/readme.md

```diff
@@ -0,0 +1,20 @@
+# rad
+
+The Rad Macro Pad is a USB-C,  Macro Pad with 11 keys powered by QMK
+
+* Keyboard Maintainer: Anubhav Dhiman 
+* Hardware Supported: ATmega32U4
+* Hardware Availability: Interest Check
+* 
+Make example for this keyboard (after setting up your build environment):

```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```